meinberlin/assets/scss/components/_dialogue_box.scss

```diff
@@ -25,8 +30,17 @@
     right: 0;
 }
 
+.dialogue-box__background-right-xl {
+    display: none;
+
+    @media screen and (min-width: $wrapper-width + 50rem) {
```

Comment:
This should also be breakpoint-lg now?

Reply:
that isn't needed anymore at all but sweet, will correct both and merge :) danke!